lib/utils/RequestHandler.js

```diff
@@ -59,11 +59,20 @@ class RequestHandler {
             throw new RuntimeError(101)
         }
 
+        // Build query string based on queryParams in defaultOptions
+        var queryString = ''
+        if (this.defaultOptions.queryParams !== undefined) {
```

Comment:
Instead of checking for being undefined let's do a typecheck to make sure queryParams is an object.